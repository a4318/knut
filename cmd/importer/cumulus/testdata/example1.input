Verbucht am,Beschreibung,Gutschrift CHF,Belastung CHF
"",Saldovortrag letzte Rechnung,,1'234.56
04.09.2020,Ihre LSV-Zahlung - Besten Dank,1'234.56,
Einkaufs-Datum,Verbucht am,Beschreibung,Gutschrift CHF,Belastung CHF
22.08.2020,24.08.2020,Desc0,,12.34
09.09.2020,10.09.2020,Desc1,,1'233.45
"",,"FXComment1",,
Verbucht am,Beschreibung,Gutschrift CHF,Belastung CHF
23.09.2020,Rundungskorrektur,0.02,