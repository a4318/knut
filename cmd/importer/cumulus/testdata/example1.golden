2020-08-22 "Desc0"
Liabilities:Cumulus Expenses:TBD             12.34 CHF

2020-09-09 "Desc1 FXComment1"
Liabilities:Cumulus Expenses:TBD           1233.45 CHF

2020-09-23 "Rundungskorrektur"
Expenses:TBD        Liabilities:Cumulus       0.02 CHF

